From 2bc3c5e92d4789146548e8de4d520cd191994e1c Mon Sep 17 00:00:00 2001
From: Marco Martin <notmart@gmail.com>
Date: Tue, 11 Feb 2020 16:41:26 +0100
Subject: unlock widgets

5.18 doesn't offer a way anymore from the gui
so unlock them if they're locked
BUG:417424
---
 desktoppackage/contents/updates/unlock_widgets.js | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 desktoppackage/contents/updates/unlock_widgets.js

diff --git a/desktoppackage/contents/updates/unlock_widgets.js b/desktoppackage/contents/updates/unlock_widgets.js
new file mode 100644
index 0000000..c332270
--- /dev/null
+++ desktoppackage/contents/updates/unlock_widgets.js
@@ -0,0 +1,2 @@
+
+__AppInterface.locked = false;
-- 
cgit v1.1

